app/lib/Plugins/SearchEngine/SqlSearch2.php

```diff
@@ -607,7 +607,7 @@ private function _processQueryPhrase(int $subject_tablenum, $query) {
 			
 				$tc = sizeof($temp_tables);
 				$qr_res = $this->db->query("
-					INSERT INTO {$temp_table}
+					INSERT IGNORE {$temp_table}
```

Comment:
I didn't know that this would work! I thought you'd need the `INTO` as well but clearly you've managed to run those queries on dev.

Reply:
I tried both ways with no change in output but i can add the `into` to be extra safe